### ISSUE TITLE
Syntax not working, am I doing something wrong?

### ISSUE THREAD

**Issue by Boolyman** [MaxNeedsSnacks/Interactio#58]
Hey there.  I have placed the below file into the `kubejs/data/inworldcrafting/recipes` folder but the recipe does nothing when I try it in world.  I have ran /kubejs errors and it is not telling me that there is any problem loading the recipe or the syntax.

{
  "type": "interactio:fluid_fluid_transform",
  "items": [
    {
      "item": "item:rustic_resources:charcoalclump",
      "count": 1,
    }
  ],
  "input": {
    "fluid": "water"
  },
  "output": {
              "fluid": "immersiveengineering:creosote__oil"
        }
}

**Comment by MaxNeedsSnacks** [comment]
Your item input's incorrect: `item:rustic_resources:charcoalclump` should be just `rustic_resources:charcoalclump` ^^

**Comment by Boolyman** [comment]
I have changed that.  And removed the incorrect "__oil" from the fluid ID.  I have quadruple checked all the IDs, and compared this syntax to the syntax used in another pack.  

There is a message on the mod download page stating that if you are on 1.15 MixinBootstrap is required.  Does that apply to 1.16.5 as well?  I have tried it with and without that as a dependency, and it still doesnt work.

I have also checked the log and there is no mention of any errors with interactio.

This is the exact syntax I am now using:

{
  "type": "interactio:fluid_fluid_transform",
  "items": [
    {
      "item": "rustic_resources:charcoalclump",
      "count": 1,
    }
  ],
  "input": {
    "fluid": "water"
  },
  "output": {
              "fluid": "immersiveengineering:creosote"
        }
}
